.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature Request
+description: Suggest an idea to enhance the current functionality of PyCytominer.
+title: "FeatRequest: "
+labels: [enhancement]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Feature type
+      description: Please check the feature type you would like to propose.
+      options:
+        - label: >
+            Add new functionality
+        - label: >
+            Change existing functionality
+  - type: textarea
+    id: description
+    attributes:
+      label: General description of the proposed functionality
+      description: >
+        Please *concisely* describe what the proposed functionality is, if it is due to an existing issue, and how it will improve PyCytominer.
```

Comment:
```suggestion
        Please provide a brief description of the proposed functionality, including whether it addresses an existing issue, and explain how it will enhance PyCytominer."
```

Reply:
👍